types/datatables.net-fixedcolumns/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for datatables.net-fixedcolumns 3.2
+// Project: https://datatables.net
+// Definitions by: Konstantin Kuznetsov <https://github.com/Arik-neKrol>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/// <reference types="jquery" />
+/// <reference types="datatables.net"/>
+
+declare namespace DataTables {
+    interface Settings {
+        /*
+         * FixedColumns extension options
+         */
+        fixedColumns?: boolean | FixedColumnsSettings;
+    }
+
+    interface FixedColumnsSettings {
+        /*
+         * Row height matching algorithm to use
+         *
+         * The algorithm to use. This can be one of (see below for full description):
+         * 'none' | 'semiauto' | 'auto'
+         */
+        heightMatch?: string;
```

Comment:
You could use a string literal type to improve the dev experience:
`heightMatch?: 'none' | 'semiauto' | 'auto';`

Reply:
Done. I created another PR with this change